spec/services/search_spec.rb

```diff
@@ -68,7 +68,7 @@
   end
 
   it 'search by fts partials (name)' do
-    res = MR::Search.new(fts: 'philos').run
+    res = MR::Search.new(fts: 'philosopher').run
```

Comment:
the idea of this test is to match a partial string, not a full one. So we have to check if the old test was not passing due a change on scoring (changed the indexed terms, so the tf-idf scoring reduces) or if the partials broke. Maybe use a slightly bigger substring like 'philosop' and check if it matches (since it gives a bigger number of trigrams matched).